pkg/datastore/commandstore.go

```diff
@@ -34,22 +34,12 @@ func (c *commandCollection) Factory() Factory {
 	}
 }
 
-var (
-	CommandHandledUpdater = func(status model.CommandStatus, metadata map[string]string, handledAt int64) func(*model.Command) error {
-		return func(cmd *model.Command) error {
-			cmd.Status = status
-			cmd.Metadata = metadata
-			cmd.HandledAt = handledAt
-			return nil
-		}
-	}
-)
-
 type CommandStore interface {
 	Add(ctx context.Context, cmd *model.Command) error
 	Get(ctx context.Context, id string) (*model.Command, error)
 	List(ctx context.Context, opts ListOptions) ([]*model.Command, error)
-	UpdateCommand(ctx context.Context, id string, updater func(piped *model.Command) error) error
+	MarkAsTimeOut(ctx context.Context, id string) error
```

Comment:
`UpdateStatus` to be unified.

Reply:
I feel you 👍 But there is one thing that concerns me, this `MarkAsTimeout` interface only be used to make the status of the command entity to `CommandStatus_COMMAND_TIMEOUT`, not the other kind. Just realized that this `c.Status` field be touched by both `ops` and `piped` at the same time, it's time for us to move this orphancommandcleaner job to other place than ops 🤔

Reply:
I get your point, just remove this `MarkAsTimeout` and use only once `UpdateStatus` interface in all cases. PTAL when you have time.